packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -32,14 +38,18 @@
   @include bpk-box-shadow-xl;
   @include bpk-border-radius-sm;
 
-  &--full-screen {
+  &--full-screen-mobile {
     @include bpk-breakpoint-mobile {
-      margin: 0;
-      border-radius: 0;
-      box-shadow: none;
+      @include fullscreen;
     }
   }
 
+  &--full-screen {
+    min-width: 100%;
```

Comment:
I think this should be `max-width: none;` instead (as this is the rule you are overriding :) )

Reply:
done